ui/imports/shared/views/ProfileDialogView.qml

```diff
@@ -428,9 +397,9 @@ Pane {
                 StatusMenu {
                     id: moreMenu
                     width: 230
-
+                    // FIXME: raly on contactRequestState !!!
```

Comment:
Still valid here?

Reply:
I forgot to move it to to draft, still waiting status-go changes